doc/neps/nep-0037-array-module.rst

```diff
@@ -71,6 +67,19 @@ There are two broad ways in which NEP-18 has fallen short of its goals:
      a separate ``np.duckarray`` function, but this still does not resolve how
      to cast one duck array into a type matching another duck array.
 
+Other issues of lesser importance include:
```

Comment:
I'm happy to split these out, but I'd call still call them "maintainability concerns" (my original heading) rather than just "issues", because both entail maintaining additional code.

Reply:
Also, for what it's worth, I'm not sure I'd say these were exactly "lesser concerns" for JAX. It's was really the whole picture that was not compelling:

- Some loss of backwards incompatibility
- Additional complex/unintuitive code
- Unclear how many users would actually benefit

Reply:
Okay, I'll change it back to "maintainability concerns". The main purpose of the changes is to have some more clarity on real issues that are potential blockers, and things that are maybe annoying but easy to work around (the worst one here is a "hey how does Dask do that, let's copy over that 30 lines of code").

Reply:
done